sdk/core/azure-core-amqp/pom.xml

```diff
@@ -14,7 +14,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>azure-core-amqp</artifactId>
-  <version>1.5.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-amqp;current} -->
+  <version>1.5.0</version> <!-- {x-version-update;com.azure:azure-core-amqp;current} -->
```

Comment:
This should be against a release branch created from the commit id we used to release core.